entries/jQuery.merge.xml

```diff
@@ -19,6 +19,7 @@ var newArray = $.merge([], oldArray);
     </code></pre>
     <p>This shortcut creates a new, empty array and merges the contents of oldArray into it, effectively cloning the array.</p>
     <p>Prior to jQuery 1.4, the arguments should be true Javascript Array objects; use <code>$.makeArray</code> if they are not.</p>
+    <p>Because <code>jQuery.merge()</code> is capable of merging array-like objects it will correct the <code>length</code> property.</p>
```

Comment:
Missing a comma before "it". But this note might be better placed near "is destructive" anyway, something like "It alters the <code>length</code> and numeric index properties of the first parameter to include items from the second."